packages/clay-card/fixtures/ClayCheckboxWithState.tsx

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
@matuzalemsteles how do you feel about these components in "fixtures"? It seems like a strange way to isolate components, and sort of ends up being its own code base to maintain. What if instead we tried to keep examples like this in the story itself, that way its easily seen from the "source" tab on storybook.

Reply:
Yeah I agree to keep the fixtures inside the Storybook people will want to see those examples over there, something we do not want to mask.

Reply:
I think some of this will also be solved by our implementations of more complex versions of the component.

Reply:
Removed!